### DISCUSSION TITLE
The 'Full Stack Course' Query

### DISCUSSION THREAD

**Discussion by dyrgutt** [notJust-dev/notjustdev#213] [General]
Three things, there are quite a few accidental mistakes on your website where the wrong video's appear for different titles in your DEVember series:

notjust.dev/projects/DEVember/day9 [Authentication in React Native using AWS Amplify V6] == WeatherAPP Video [x]
notjust.dev/projects/DEVember/day10 [Fingerprint and FaceID] == WeatherAPP Video [x]
notjust.dev/projects/DEVember/day11 Camera App with React Native Vision Camera == [missing video]
notjust.dev/projects/DEVember/day15 [Ultimate To-Do app in React Native] == In-app Subscriptions [x] 

Secondly, when you click the 'Next or Back Part' button at the bottom of these screens, the newsletter form 'class="seva-form formkit-form"' will replicate itself equal to the amount of times you've clicked those buttons (...gets quite unruly).

Thirdly, I wish to signup/purchase your 'The Full Stack Pkg' - though noticed the course appears to be one I have already done after watching one of your live streams and receiving a Free 2-Day access of building 'notjustfriends'; is it essentially the same, or is there further and significant development beyond the free course version? I would love to sign-up and expediate my progression, but i'm concerned it's what i've already tackled? Anyway, just paying a smidge back for all your awesome effort/videos etc., regarding the above accidental quirks.

Keep up the great work and thanks in advance for your reply to my course query :)

**Comment by VadimNotJustDev** [comment]
Hey, First of all thank you very much for pointing out those mistakes. I have solved them.

Still have to deal with the sign up form. 

About the course, the project and the depth of the materials you will learn in the course do not even come close to what we do on youtube and in the masterclas. It covers all the aspects of building a full stack app, goes in depth into the backend concepts and helps you build a strong foundation. It has 10 modules and over 40h of content.

**Comment by Prrizm** [comment]
Hi there, is your code for the onlyfans clone app or tinder clone app on your github profile include the front end code to match? Sorry I'm new to coding I'm trying to use your clone apps as a template for one of mine.

Thanks and I look forward to hearing from you :)
